src/plugin/urlfilter-model/src/java/org/apache/nutch/urlfilter/model/NBClassifier.java

```diff
@@ -0,0 +1,234 @@
+/**
```

Comment:
The name of this file should be NaiveBayesClassifier as the NBClassifier is highly ambiguous